android/defaults/deps.nix

```diff
@@ -450,15 +450,15 @@
   { artifactId = "apache";
     groupId = "org.apache";
     version = "13";
-    repo = "https://central.maven.org/maven2/";
+    repo = "https://repo.maven.apache.org/maven2/";
```

Comment:
I think this can also be changed to repo1.maven.org

Reply:
https://repo1.maven.org/maven2/org/apache/apache/13/

Reply:
Could be, I just picked the first server that actually worked.